### ISSUE TITLE
Nameservers not returned for .dk domains

### ISSUE THREAD

**Issue by emilstahl** [turbot/steampipe-plugin-whois#16]
**Describe the bug**
Nameservers not returned for .dk domains. The domain is returned.

**Steampipe version**
Example:  0.7.3

**Plugin version)**
Example: 0.2.0

**To reproduce**
Run:
```
> select name_servers
from
  whois_domain
where
  domain in ('emilstahl.dk');
+------------------+
| name_servers     |
+------------------+
| ["emilstahl.dk"] |
+------------------+
```

**Expected behavior**
As a normal whois query returns:
```Nameservers
Hostname:             ns1.simply.com
Hostname:             ns2.simply.com
Hostname:             ns3.simply.com
Hostname:             ns4.simply.com
```

The output should be:
` ["ns1.simply.com","ns2.simply.com","ns3.simply.com","ns4.simply.com"]`

**Comment by e-gineer** [comment]
Thanks @emilstahl for using Steampipe and reporting this issue 🙂

interesting that it seems to return the domain as it’s own name server. I suspect this is actually an issue with the Whois parser library we are using, but will need to do more testing to see why.

WHOIS data tends to vary a lot by TLD so perhaps there is something different about the .dk data.

**Comment by emilstahl** [comment]
Yeah, the DK whois format/server is custom. 👍

**Comment by emilstahl** [comment]
@e-gineer  Can you remove the stale label?

**Comment by cbruno10** [comment]
Hey @emilstahl , sorry for the radio silence on this, we'll be taking a look at this issue in the next week or so, and will post any updates here.

**Comment by cbruno10** [comment]
@emilstahl Sorry for the long wait on this issue. I ran some tests against emilstahl.dk using the latest version of the `whois` and `whois-parser` packages (these changes are in the `update-whois-packages` branch), and it seems like the way the WHOIS data is being returned is unexpected in the `whois-parser` package, so it incorrectly pulls out the nameservers as @e-gineer had suspected.

I've opened https://github.com/likexian/whois-parser/issues/52 as a feature request for better parsing. If you have any other information, feel free to add comments to that issue as well, thanks!
